docs/governance/security-template.md

```diff
@@ -0,0 +1,142 @@
+
+Once your project's security vulnerability disclosure policy document is
+published, place a copy of it into each of your project's repositories, or
+create a `SECURITY.md` file in each of your project's repositories that links
+back to the primary document.
+
+[OpenWallet Foundation's security vulnerability disclosure policy]: https://tac.openwallet.foundation/governance/security/
+
+## About this Document
+This document document defines how security vulnerability reporting is handled
+in this project. The approach aligns with the [OpenWallet Foundation's security
+vulnerability disclosure policy]. Please review that document to understand
+the basis of the security reporting for this project
+
+This policy borrows heavily from the recommendations of the OpenSSF
+Vulnerability Disclosure working group. For up-to-date information on the latest
+recommendations related to vulnerability disclosures, please visit the [GitHub
+of that working group](https://github.com/ossf/wg-vulnerability-disclosures).
+
+If you are already familiar with what a security vulnerabulity disclosure policy
+is and are ready to report a vulnerability, please jump to [Report
+Intakes](#report-intakes).
+
+## What Is a Vulnerability Disclosure Policy?
+
+No piece of software is perfect. All software (at least, all software of a
+certain size and complexity) has bugs. In open source development, members of
+the community or the public find bugs and report them to the project. A
+vulnerability disclosure policy explains how this process functions from the
+perspective of the project.
+
+This vulnerability disclosure policy explains the rules and guidelines for
+this project. It is intended to act as both a reference for
+outsiders–including both bug reporters and those looking for information on the
+project’s security practices–as well as a set of rules that maintainers and
+contributors have agreed to follow.
+
+## Report Intakes
+
+This project has the following ways to submit security
+vulnerabilities. While the security team members will do their best to
+respond to bugs disclosed in all possible ways, it is encouraged for bug
+finders to report through the following approved channels:
+
+- Email the [OpenWallet Foundation's security list]: To report a security issue, please send an email with the name of the project/repository, a description of the issue, the steps you took to create the issue, affected versions, and if known, mitigations. If in triaging the email, the security team determines the issue may be a security vulnerability, a [GitHub security vulnerability report] will be opened.
+- Open a [GitHub security vulnerability report]: Open a draft security advisory
+on the "Security" tab of this GitHub repository. See [GitHub Security Advisories](#github-security-advisories) to learn more about the security infrastructure in GitHub.
+
+[OpenWallet Foundation's security list]: mailto:security@lists.openwallet.foundation
```

Comment:
It looks like we aim for a centralized way to collect the reports, which is not a bad idea in its principles. This will keep the foundation informed.

On the other hand, for security topics, the sharing of information should follow the need-to-know principles, raising the following questions:
1. How broad is the audience of this list? Do all its participants need to know the vulnerabilities of particular repositories?
2. How will the information securely reach the repository's security team?

Reply:
There is a potential concern here of providing too much information. If we do not want a "central" mechanism, we could limit security reports to just utilizing the GitHub reporting mechanism. @ryjones @hartm I would appreciate your thoughts on this.

Reply:
I expect that, unlike Hyperledger's structure, our repositories will not consist of homogeneous teams with guaranteed communication channels between them. Additionally, I'm unsure if the maintainers of a particular repository would feel comfortable having their vulnerabilities exposed to an audience not controlled by them, before they resolve them. (Please correct me if I'm wrong.)

That said, I can't determine if we need or want the "central" mechanism. Therefore, I would support just utilizing GitHub's mechanism for the moment.

Reply:
yes, I agree, use the github stuff

Reply:
Using the github stuff is fine, as long as it is well-documented how to do this.  If we did it over again for Hyperledger, we probably would just use this and not bother with an email list (although an email list does offer some advantages).

"Additionally, I'm unsure if the maintainers of a particular repository would feel comfortable having their vulnerabilities exposed to an audience not controlled by them, before they resolve them. (Please correct me if I'm wrong.)". This is usually not a problem in practice, since the security list is typically small and consists of people who have a lot of reputation to lose if they were to abuse something like this.